catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-converter/pom.xml

```diff
@@ -184,17 +184,17 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.62</minimum>
+                                            <minimum>0.57</minimum>
                                         </limit>
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.40</minimum>
+                                            <minimum>0.33</minimum>
                                         </limit>
                                         <limit>
                                             <counter>COMPLEXITY</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.34</minimum>
+                                            <minimum>0.18</minimum>
```

Comment:
This drop in coverage is somewhat significant; is there a reason behind it?

Reply:
Turns out there was some dead code that it was picking up. Removing increased our coverage.